incubator/group/integration_test.go

```diff
@@ -112,7 +112,108 @@ func TestCreateGroupScenario(t *testing.T) {
 			assert.True(t, app.GroupKeeper.HasGroup(ctx, resp.Data))
 		})
 	}
+}
+
+func TestCreateGroupAccountScenario(t *testing.T) {
+	app, ctx := createTestApp(false)
+	myKey, _, myAddr := types.KeyTestPubAddr()
+	myAccount := app.AccountKeeper.NewAccountWithAddress(ctx, myAddr)
+	app.AccountKeeper.SetAccount(ctx, myAccount)
+
+	_, _, otherAddr := types.KeyTestPubAddr()
+
+	balances := sdk.NewCoins(sdk.NewInt64Coin("atom", 10000))
+	require.NoError(t, app.BankKeeper.SetBalances(ctx, myAddr, balances))
+
+	myGroupID, err := app.GroupKeeper.CreateGroup(ctx, myAddr, nil, "integration test")
+	require.NoError(t, err)
+
+	fee := types.NewTestStdFee()
+	specs := map[string]struct {
```

Comment:
Can you add a spec with custom `Threshold` and `Timeout`s

Reply:
They are covered in unit test: https://github.com/regen-network/cosmos-modules/pull/52/files#diff-7b4842d86e976eaae9d5f8ac29447b16R192

Integration tests setup a full stack and are quite slow compared to the unit tests. That's why I try to keep them to the bare minimum.